packages/platform-sdk-ada/src/dto/signed-transaction.ts

```diff
@@ -20,6 +21,10 @@ export class SignedTransactionData
 		return BigNumber.make(this.signedData.fee);
 	}
 
+	public timestamp(): DateTime {
+		return this.signedData.timestamp;
```

Comment:
```suggestion
		return DateTime.make(this.signedData.timestamp);
```

There's cases where we restore an instance from JSON which means the timestamp could be a string. This function will simply return the existing instance if the value is already DateTime.